method.js

```diff
@@ -0,0 +1,14 @@
+async function sayJoke(apiUrl, jokeId) {
+  const response = await fetch(apiUrl)
+  const json = await response.json()
+  for(let joke of json.jokes){
+      if (joke.id == jokeId){
+        return {saySetup: () => joke.setup,
+               sayPunchLine: () => joke.punchLine
+              };
+   }
+    throw new Error (`No jokes found id: ${jokeId}`);
+  }).catch(error => {
```

Comment:
I didn't find any `try` block, to have this `catch` here

Reply:
Done